sql/core/src/test/scala/org/apache/spark/sql/vectorized/ArrowColumnVectorSuite.scala

```diff
@@ -15,15 +15,14 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.execution.vectorized
+package org.apache.spark.sql.vectorized
```

Comment:
It's fine, just not sure why this is necessary. ColumnVector is in org.apache.spark.sql.vectorized, but ColumnVectorSuite is also in org.apache.spark.sql.execution.vectorized.

There seems no other test suites under org.apache.spark.sql.vectorized?

Reply:
Thanks. We can move other tests in a separate PR